asm/macros.inc

```diff
@@ -1 +1,7 @@
+	.include "asm/macros/misc.inc"
```

Comment:
Minor but keeping the alignment consistent would be nice

Reply:
Sure! I have no idea why it's aligned like that. :D